build/common.sh

```diff
@@ -1427,3 +1437,40 @@ function kube::release::gcs::publish() {
     return 1
   fi
 }
+
+# ---------------------------------------------------------------------------
+# Docker Release
+
+# Releases all docker images to a docker registry specified by KUBE_DOCKER_REGISTRY
+# using tag KUBE_DOCKER_IMAGE_TAG.
+#
+# Globals:
+#   KUBE_DOCKER_REGISTRY
+#   KUBE_DOCKER_IMAGE_TAG
+# Returns:
+#   If new pushing docker images was successful.
+function kube::release::docker::release() {
+  local binaries=(
+    "kube-apiserver"
+    "kube-controller-manager"
+    "kube-scheduler"
+    "kube-proxy"
+  )
+
+  local platforms=(
+    "linux-amd64"
```

Comment:
@fgrzadkowski I'm working on it. 
And yes, it's possible to `dynamically` cross-compile ARM binaries on `amd64` hosts.
It's even possible to build ARM docker images on `amd64`, like `hyperkube` for example.
Inspiration: https://resin.io/blog/building-arm-containers-on-any-x86-machine-even-dockerhub/

I was so happy when I got this working: https://gist.github.com/luxas/e319b314f22b18243b1d
I have (locally) integrated this gist into `kube-build` and it worked even on `ARMv6` (e.g. `hyperkube` as it is pushed now, only works on `ARMv7`)
So I'm gonna upload a PR with my work when it's ready.